### PR TITLE
Lockfile post 19.2

### PR DESCRIPTION
**Pull request by theHamsta**
It's a lot faster to do this in a automated way.

@sogaiou could you create a PR when you merge your 0.19.X branch?

### CODE REVIEW

**Comment by theHamsta** [comment]
@kyazdani42 It still expects the missing check.

File: .github/workflows/update-parsers-pr.yml [hunk 1]
```diff
@@ -27,6 +27,8 @@ jobs:
           ln -s $(pwd) ~/.local/share/nvim/site/pack/nvim-treesitter/start
 
       - name: Update parsers
+        env:
+          SKIP_LOCKFILE_UPDATE_FOR_LANGS: clojure
```
Comment (theHamsta) [inline .github/workflows/update-parsers-pr.yml:31]: @sogaiu could you remove this as soon as you merge your 0.19.X branch?

**Review by kyazdani42 — APPROVED** [review]
should we merge then ? I should definitely look at the ga checks to understand these things better x)

**Comment by theHamsta** [comment]
I'd merge. Those required settings are no where in our repo itself. The should be in repo settings for owners (what is required to get something merged)

**Comment by theHamsta** [comment]
@kyazdani42 I suppose that the master branch has some rules in the main repo

image

**Comment by theHamsta** [comment]
@stsewd sorry, for complaining. It was only difficult for me and a lot of work for you to keep a manual list

A manual list didn't help me at all

- since it was incorrect because we are humans
- it did not help me at all. I run the update script and test to see which parsers required updates (every failing parser was not updated). I had to check all parsers a again just to see that I already did it

**Pull request merged**